packages/plugin-commands-env/test/env.test.ts

```diff
@@ -133,3 +133,81 @@ test('it re-attempts failed downloads', async () => {
     nock.cleanAll()
   }
 })
+
+describe('env remove', () => {
+  test('fail if --global is missing', async () => {
+    tempDir()
+
+    await expect(
+      env.handler({
+        bin: process.cwd(),
+        global: false,
+        pnpmHomeDir: process.cwd(),
+        rawConfig: {},
+      }, ['remove', 'lts'])
+    ).rejects.toEqual(new PnpmError('NOT_IMPLEMENTED_YET', '"pnpm env use <version>" can only be used with the "--global" option currently'))
+  })
+
+  test('fail if can not resolve Node.js version', async () => {
+    tempDir()
+
+    await expect(
+      env.handler({
+        bin: process.cwd(),
+        global: true,
+        pnpmHomeDir: process.cwd(),
+        rawConfig: {},
+      }, ['rm', 'non-existing-version'])
+    ).rejects.toEqual(new PnpmError('COULD_NOT_RESOLVE_NODEJS', 'Couldn\'t find Node.js version matching non-existing-version'))
+  })
+
+  test('fail if trying to remove version that is not installed', async () => {
+    tempDir()
+
+    const nodeDir = node.getNodeVersionsBaseDir(process.cwd())
+
+    await expect(
+      env.handler({
+        bin: process.cwd(),
+        global: true,
+        pnpmHomeDir: process.cwd(),
+        rawConfig: {},
+      }, ['remove', '16.4.0'])
+    ).rejects.toEqual(new PnpmError('ENV_NO_NODE_DIRECTORY', `Couldn't find Node.js directory in ${path.resolve(nodeDir, '16.4.0')}`))
+  })
+
+  test('install and remove Node.js by exact version', async () => {
+    tempDir()
+
+    const configDir = path.resolve('config')
+
+    await env.handler({
+      bin: process.cwd(),
+      configDir,
+      global: true,
+      pnpmHomeDir: process.cwd(),
+      rawConfig: {},
+    }, ['use', '16.4.0'])
+
+    const opts = {
+      env: {
+        [PATH]: `${process.cwd()}${path.delimiter}${process.env[PATH] as string}`,
```

Comment:
I don't understand how this test works. Even if you remove the node.js from home path, won't execa pick the system installed node.js?

Reply:
I'm sorry, I didn't have the capacity to respond yesterday.

So, you're correct, it should have picked other (in my case nvm) installed versions. I was going to restrict path to only the current folder, but I see that you've done that already.

I think it's worth mentioning that execa, for some reason (I didn't get the chance to investigate further yet) tries to resolve the "node" in the temp folder first, I'm not sure why that's happening exactly.

Anyway, thanks for merging!